src/forms/login-popup/index.jsx

```diff
@@ -83,16 +83,16 @@ const LoginForm = () => {
   }, []);
 
   useEffect(() => {
-    if (thirdPartyAuthApiStatus === COMPLETE_STATE) {
+    if (thirdPartyAuthApiStatus === COMPLETE_STATE && accountActivation === null) {
```

Comment:
Could you please explain why this condition `accountActivation === null` was added?

Reply:
Because we want to set the focus on accountActivation when accountActivation exist otherwise we want to set it on SSO button and email.